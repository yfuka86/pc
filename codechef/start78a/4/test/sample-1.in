9
30 1
56 4
364 2
125 3
37662730 3
41872528 4
73170084 8
90032975 1
7487471 3
