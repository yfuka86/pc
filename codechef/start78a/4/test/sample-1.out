3
0
2
0
603
1583
0
9967024
3
